src/dotnet-interactive/JupyterInstallCommand.cs

```diff
@@ -75,14 +75,16 @@ private static void ComputeKernelSpecArgs(HttpPortRange httpPortRange, Directory
 
             foreach (var kernelSpec in kernelSpecs)
             {
-                var parsed = JObject.Parse(File.ReadAllText(kernelSpec.FullName));
+                var newKernelSpec = JsonDocument.Parse(File.ReadAllText(kernelSpec.FullName)).RootElement.EnumerateObject().ToDictionary(p => p.Name, p => p.Value);
+                
+                var argv = newKernelSpec["argv"].EnumerateArray().Select(e => e.GetString()).ToList();
 
-                var argv = parsed["argv"].Value<JArray>();
+                argv.Add( "--http-port-range");
+                argv.Add($"{httpPortRange.Start}-{httpPortRange.End}");
 
-                argv.Insert(argv.Count - 1, "--http-port-range");
-                argv.Insert(argv.Count - 1, $"{httpPortRange.Start}-{httpPortRange.End}");
+                newKernelSpec["argv"] = JsonSerializer.SerializeToElement(argv);
 
-                File.WriteAllText(kernelSpec.FullName, parsed.ToString(Newtonsoft.Json.Formatting.Indented));
+                File.WriteAllText(kernelSpec.FullName, JsonSerializer.Serialize(newKernelSpec));
```

Comment:
Should this be opinionated about which serialization settings are used? Are we contract-testing this?

Reply:
this is jsut editing a json document ( the kernel spec file) leaving all untouched and jsut patching the argv array, the types are left as jsonelement all the way

Reply:
makes sense, added contract pinning test